IFC4x3/Constants/w/WATER_0SG7sVKEfA0OS6iZfD6HgM.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="WATER_0SG7sVKEfA0OS6iZfD6HgM" Name="WATER" UniqueId="1c407d9f-50ea-4a01-8706-b23a4d191a96">
+	<Documentation>Representation of the concept of an identified discrete water filled geological or surface feature including lakes, rivers and seas.</Documentation>
+</DocConstant>
+
```

Comment:
IfcGeoScienceFeature from IfcTunnel will be covering this in another way.

Reply:
Agreed.